Source/Editor/Editor.cpp

```diff
@@ -519,6 +519,12 @@ bool Editor::Init()
         return true;
     }
 
+    // Skip compilation if play on start
+    if (CommandLine::Options.Game.HasValue())
```

Comment:
Does it work without compilation skipping? Because devs can already use `-skipcompile` arg (added by default to C++ projects debug cmd line) or disable initial compilation in Editor options.

Reply:
Ah yes ofc !